Gruntfile.js

```diff
@@ -1,279 +1,303 @@
 // jscs:disable maximumLineLength
```

Comment:
I'd get rid of Grunt entirely in another PR.

Reply:
Yes